Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -397,21 +401,46 @@ WarpX::computePhiCartesian (const amrex::Vector<std::unique_ptr<amrex::MultiFab>
         }
     }
 
-    // set the boundary potential values if needed
+#ifndef AMREX_USE_EB
     setPhiBC(phi, dirichlet_flag, phi_bc_values_lo, phi_bc_values_hi);
```

Comment:
In my testing I haven't had any issues with setting the domain boundary potentials while also using an embedded boundary. But this requires the `setPhiBC` function to still be called. Do you anticipate a scenario where this would be problematic and therefore only call `setPhiBC` if EB's are not used?

Reply:
OK, sounds good, I will update the PR accordingly.

Reply:
Thanks!